diff -Nru gperf-2.7/doc/Makefile.in gperf-2.7.new/doc/Makefile.in
--- gperf-2.7/doc/Makefile.in	Tue Apr 14 23:31:45 1998
+++ gperf-2.7.new/doc/Makefile.in	Sun Sep 26 19:23:55 1999
@@ -88,23 +88,18 @@
 	cd $(srcdir) && $(TEXI2HTML) -expandinfo -number -split_chapter gperf.texi
 
 
-install : all force
-	if [ ! -d $(infodir) ] ; then mkdir $(infodir) ; fi
-	$(INSTALL_DATA) $(srcdir)/gperf.info $(infodir)/gperf.info
-	if [ ! -d $(mandir) ] ; then mkdir $(mandir) ; fi
-	if [ ! -d $(man1dir) ] ; then mkdir $(man1dir) ; fi
-	$(INSTALL_DATA) $(srcdir)/gperf.1 $(man1dir)/gperf.1
-	if [ ! -d $(dvidir) ] ; then mkdir $(dvidir) ; fi
-	$(INSTALL_DATA) $(srcdir)/gperf.dvi $(dvidir)/gperf.dvi
-	if [ ! -d $(htmldir) ] ; then mkdir $(htmldir) ; fi
-	$(INSTALL_DATA) $(srcdir)/gperf.html $(htmldir)/gperf.html
+install : all force installdirs
+	$(INSTALL_DATA) $(srcdir)/gperf.info $(DESTDIR)$(infodir)/gperf.info
+	$(INSTALL_DATA) $(srcdir)/gperf.1 $(DESTDIR)$(man1dir)/gperf.1
+	$(INSTALL_DATA) $(srcdir)/gperf.dvi $(DESTDIR)$(dvidir)/gperf.dvi
+	$(INSTALL_DATA) $(srcdir)/gperf.html $(DESTDIR)$(htmldir)/gperf.html
 
 installdirs : force
-	if [ ! -d $(infodir) ] ; then mkdir $(infodir) ; fi
-	if [ ! -d $(mandir) ] ; then mkdir $(mandir) ; fi
-	if [ ! -d $(man1dir) ] ; then mkdir $(man1dir) ; fi
-	if [ ! -d $(dvidir) ] ; then mkdir $(dvidir) ; fi
-	if [ ! -d $(htmldir) ] ; then mkdir $(htmldir) ; fi
+	if [ ! -d $(DESTDIR)$(infodir) ] ; then mkdir -p $(DESTDIR)$(infodir) ; fi
+	if [ ! -d $(DESTDIR)$(mandir) ] ; then mkdir -p $(DESTDIR)$(mandir) ; fi
+	if [ ! -d $(DESTDIR)$(man1dir) ] ; then mkdir -p $(DESTDIR)$(man1dir) ; fi
+	if [ ! -d $(DESTDIR)$(dvidir) ] ; then mkdir -p $(DESTDIR)$(dvidir) ; fi
+	if [ ! -d $(DESTDIR)$(htmldir) ] ; then mkdir -p $(DESTDIR)$(htmldir) ; fi
 
 uninstall : force
 	$(RM) $(infodir)/gperf.info
diff -Nru gperf-2.7/src/Makefile.in gperf-2.7.new/src/Makefile.in
--- gperf-2.7/src/Makefile.in	Thu Apr 16 13:22:10 1998
+++ gperf-2.7.new/src/Makefile.in	Sun Sep 26 19:21:35 1999
@@ -119,14 +119,14 @@
 	$(CXX) $(CXXFLAGS) $(CPPFLAGS) -c $(srcdir)/version.cc
 
 install : all force
-	if [ ! -d $(bindir) ] ; then mkdir $(bindir) ; fi
-	$(INSTALL_PROGRAM) $(TARGETPROG) $(bindir)/$(TARGETPROG)
+	if [ ! -d $(DESTDIR)$(bindir) ] ; then mkdir -p $(DESTDIR)$(bindir) ; fi
+	$(INSTALL_PROGRAM) $(TARGETPROG) $(DESTDIR)$(bindir)/$(TARGETPROG)
 
 installdirs : force
-	if [ ! -d $(bindir) ] ; then mkdir $(bindir) ; fi
+	if [ ! -d $(DESTDIR)$(bindir) ] ; then mkdir -p $(DESTDIR)$(bindir) ; fi
 
 uninstall : force
-	$(RM) $(bindir)/$(TARGETPROG)
+	$(RM) $(DESTDIR)$(bindir)/$(TARGETPROG)
 
 check : all
 
