--- gperf-3.0.1/doc/Makefile.in.orig	2003-06-02 13:42:02.000000000 +0200
+++ gperf-3.0.1/doc/Makefile.in	2003-10-14 00:53:19.827335328 +0200
@@ -64,7 +64,7 @@
 
 VPATH = $(srcdir)
 
-all : info ps pdf html
+all : info
 
 
 info : $(srcdir)/gperf.info
@@ -116,8 +116,8 @@
 #	$(INSTALL_DATA) $(srcdir)/gperf.ps $(DESTDIR)$(psdir)/gperf.ps
 #	$(MKINSTALLDIRS) $(DESTDIR)$(pdfdir)
 #	$(INSTALL_DATA) $(srcdir)/gperf.pdf $(DESTDIR)$(pdfdir)/gperf.pdf
-	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
-	$(INSTALL_DATA) $(srcdir)/gperf.html $(DESTDIR)$(htmldir)/gperf.html
+#	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
+#	$(INSTALL_DATA) $(srcdir)/gperf.html $(DESTDIR)$(htmldir)/gperf.html
 
 installdirs : force
 	$(MKINSTALLDIRS) $(DESTDIR)$(infodir)
@@ -125,7 +125,7 @@
 #	$(MKINSTALLDIRS) $(DESTDIR)$(dvidir)
 #	$(MKINSTALLDIRS) $(DESTDIR)$(psdir)
 #	$(MKINSTALLDIRS) $(DESTDIR)$(pdfdir)
-	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
+#	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
 
 uninstall : force
 	$(RM) $(DESTDIR)$(infodir)/gperf.info
@@ -133,7 +133,7 @@
 #	$(RM) $(DESTDIR)$(dvidir)/gperf.dvi
 #	$(RM) $(DESTDIR)$(psdir)/gperf.ps
 #	$(RM) $(DESTDIR)$(pdfdir)/gperf.pdf
-	$(RM) $(DESTDIR)$(htmldir)/gperf.html
+#	$(RM) $(DESTDIR)$(htmldir)/gperf.html
 
 check : all
 
