--- gperf-3.1/doc/Makefile.in.orig	2017-01-02 12:35:39.000000000 +0100
+++ gperf-3.1/doc/Makefile.in	2017-01-23 21:51:44.042712664 +0100
@@ -64,7 +64,7 @@
 
 VPATH = $(srcdir)
 
-all : info pdf html man
+all : info man
 
 
 info : $(srcdir)/gperf.info
@@ -130,8 +130,8 @@
 #	$(INSTALL_DATA) $(srcdir)/gperf.ps $(DESTDIR)$(psdir)/gperf.ps
 #	$(MKINSTALLDIRS) $(DESTDIR)$(pdfdir)
 #	$(INSTALL_DATA) $(srcdir)/gperf.pdf $(DESTDIR)$(pdfdir)/gperf.pdf
-	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
-	$(INSTALL_DATA) $(srcdir)/gperf.html $(DESTDIR)$(htmldir)/gperf.html
+#	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
+#	$(INSTALL_DATA) $(srcdir)/gperf.html $(DESTDIR)$(htmldir)/gperf.html
 
 installdirs : force
 	$(MKINSTALLDIRS) $(DESTDIR)$(infodir)
@@ -139,7 +139,7 @@
 #	$(MKINSTALLDIRS) $(DESTDIR)$(dvidir)
 #	$(MKINSTALLDIRS) $(DESTDIR)$(psdir)
 #	$(MKINSTALLDIRS) $(DESTDIR)$(pdfdir)
-	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
+#	$(MKINSTALLDIRS) $(DESTDIR)$(htmldir)
 
 uninstall : force
 	$(RM) $(DESTDIR)$(infodir)/gperf.info
@@ -147,7 +147,7 @@
 #	$(RM) $(DESTDIR)$(dvidir)/gperf.dvi
 #	$(RM) $(DESTDIR)$(psdir)/gperf.ps
 #	$(RM) $(DESTDIR)$(pdfdir)/gperf.pdf
-	$(RM) $(DESTDIR)$(htmldir)/gperf.html
+#	$(RM) $(DESTDIR)$(htmldir)/gperf.html
 
 check : all
 
